Dockerfile

```
@@ -1,11 +1,11 @@
-FROM docker.io/clojure:temurin-21-tools-deps-alpine AS jar-build
+FROM docker.io/clojure:temurin-23-tools-deps-alpine AS jar-build
 RUN apk add git
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)